lightning/src/offers/async_receive_offer_cache.rs

```diff
@@ -280,9 +292,9 @@ impl AsyncReceiveOfferCache {
 	/// to build a new offer.
 	pub(super) fn should_build_offer_with_paths(
 		&self, offer_paths: &[BlindedMessagePath], offer_paths_absolute_expiry_secs: Option<u64>,
-		duration_since_epoch: Duration,
+		cache_slot: u16, duration_since_epoch: Duration,
```

Comment:
Perhaps keep using the name `invoice_slot`? It raised the question with me whether cache_slot is something different.

Reply:
Don't offer and invoice slots correspond 1:1? Maybe just `slot_idx` then everywhere? I think it would be nice to capture that 1:1 correspondence via equal names.

Reply:
Renamed to `slot` per offline discussion